src/types/index.ts

```diff
@@ -86,10 +86,19 @@ export interface RequestSnippetSchema {
   after: string[]
 }
 
+export interface StudioFile {
```

Comment:
I don't like the name `StudioFile`, but I don't want it the type to collide with the built-in `File` type. Suggestions are welcome.